configuration.nix

```diff
@@ -11,9 +11,25 @@
     # This is not needed when deploying to a virtual box.
     #./hardware-configuration.nix
   ];
-  services.nix-bitcoin.enable = true;
-  # FIXME Install and use minimal or all modules
-  services.nix-bitcoin.modules = "all";
+  # FIXME Enable modules by uncommenting their respective line. Disable modules by commenting out their respective line.
+    # Enable this module to use the nix-bitcoin node configuration. Only disable this if you know what you are doing.
```

Comment:
Nit: Can you remove the indentation of the whole block to be on the same level as FIXME? And would also be nice to have a fixed `textwidth` (i.e. width in number of chars). The rest of the text has width 78.